changelog.md

```diff
@@ -19,6 +19,9 @@
 - Fix retry of MQTT publish frame in `vmq_bridge` (#1084).
 - Fix outgoing qos2 message id bug (#1045).
 - VerneMQ now requires Erlang/OTP 21.2 or newer to build.
+- Change `vmq_reg:publish/4` to return the number of matched subscriptions. This
```

Comment:
Could you explain how it breaks that publish API and how it could be used instead?

Reply:
as far as I understood it it is only the return value which is different. Before `ok` was returned, now `{ok, {NumLocalMatches,NumRemoteMatches}}` is returned. Maybe it would be good to mention this in the changelog... /cc @dergraf

Reply:
Done

Reply:
Looks good to me, any specific reason why this Api is not made public as part of vernemq_dev? That would be  a nice way to track changes, even if a major change is done people will easily know about it because is a public/documented Api IMHO. Pretty sure lot of people use it in their plugins as i do myself, i find this Api to be probably one of the most powerful functionalities of VerneMQ against other broker we tried.